int8 x_spd
int8 y_spd
int8 x_position
int8 y_position
int16 dist0
int16 dist1
int16 dist2
int16 dist3
int16 dist4
int16 dist5
uint8 sw_l
uint8 sw_r
uint8 enc_exec_done